rstudio/rstudio
# Remove r-base from recommended dependencies for linux

### Intent

> Remove r-base from the recommended dependencies for linux builds so we don't re-install the wrong version of R. Addresses #9880

### Approach

> Remove r-base from recommends and change it to suggests in `package/linux/CMakeLists.txt`

### Automated Tests

> NA

### QA Notes

- On Ubuntu/Debian system, install R using [instructions here](https://docs.rstudio.com/resources/install-r/). Verify R installation and version with `/opt/R/${R_VERSION}/bin/R --version`.  Then, install the RStudio debian package (.deb) with apt or apt-get. Verify that R does not get re-installed and the base R version doesn't change. 

- If installing RStudio and base R has not yet been installed, it sshould still suggest installing R

### Checklist
